lit/docs/operation/metrics.lit

```diff
@@ -118,9 +118,24 @@ metrics pipeline.
     # TYPE concourse_locks_held gauge
     concourse_locks_held{type="Batch"} 0
     concourse_locks_held{type="DatabaseMigration"} 0
+    # HELP concourse_tasks_wait_duration Elapsed time waiting for execution
+    # TYPE concourse_tasks_wait_duration histogram
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="1"} 0
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="15"} 0
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="30"} 0
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="60"} 0
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="120"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="180"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="240"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="300"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="600"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="1200"} 1
+    concourse_tasks_wait_duration_bucket{platform="linux",teamId="1",workerTags="",le="+Inf"} 1
+    concourse_tasks_wait_duration_sum{platform="linux",teamId="1",workerTags=""} 60.0001929
+    concourse_tasks_wait_duration_count{platform="linux",teamId="1",workerTags=""} 1
```

Comment:
I'm confused - where does this information appear in the docs? All I see is:

<img width="729" alt="Screen Shot 2020-08-19 at 10 05 56 AM" src="https://user-images.githubusercontent.com/26583442/90645141-9b822b80-e203-11ea-9575-b2add8f0853f.png">

Reply:
To be fair, it seems consistent with how other metrics are defined, I'm just not sure how it works tbh

Reply:
+1 that. I didn't dig into it but I'm guessing someone (@vito ?) wrote a lit plugin to parse the output of the metrics endpoint. If you bring up concourse with the `./hack/overrides/prometheus.yml` and visit http://localhost:9100/ you'll see the output in this doc. That's where I got this "data" 😁

Reply:
Yep, all the metrics docs are generated via the `PrometheusDocs` plugin function, so all you need to do is copy paste the response. The response already includes descriptions and types so I figured this would be an easier flow than hand typing the same info in two places.

Reply:
One small issue with this that I haven't seen written anywhere (maybe it should be a comment in this file?) is that not all metrics are listed in the response. There are a few metrics (e.g. the two this PR adds/updates) that don't appear in the output until some data is generated for them. So it's not always as simple as "copy and paste the entire response" because the response you're copying probably doesn't contain all metrics 😬

Reply:
Ah dang that's true.